From d9785f9dad3ba408ff3539a4fe3bcdd7db864f90 Mon Sep 17 00:00:00 2001
From: SN-Abdullah-Al-Noman <snprotectserver12@gmail.com>
Date: Sun, 11 May 2025 08:48:46 +0600
Subject: [PATCH] Disable auto add extra localversion.

---
 scripts/setlocalversion          | 1 -
 1 files changed, 1 insertion(+), 1 deletions(-)
diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 365b3c2b8..49f4f7c6f 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -176,7 +176,6 @@ else
 	# LOCALVERSION= is not specified
 	if test "${LOCALVERSION+set}" != "set"; then
 		scm=$(scm_version --short)
-		res="$res${scm:++}"
 	fi
 fi
