From d4d33ba319872f696f86efa7f405d72c4bd42b14 Mon Sep 17 00:00:00 2001
From: SN-Abdullah-Al-Noman <snprotectserver12@gmail.com>
Date: Mon, 16 Dec 2024 12:31:40 +0600
Subject: [PATCH] selinux: remove debug and enforce flag.

---
 security/selinux/Makefile | 21 +--------------------
 1 file changed, 1 insertion(+), 20 deletions(-)

diff --git a/security/selinux/Makefile b/security/selinux/Makefile
index 3c989c966..8a768d68c 100755
--- a/security/selinux/Makefile
+++ b/security/selinux/Makefile
@@ -2,27 +2,8 @@
 #
 # Makefile for building the SELinux module as part of the kernel tree.
 #
-# [ SEC_SELINUX_PORTING_COMMON
-ccflags-y := -I$(srctree)/security/selinux -I$(srctree)/security/selinux/include
 
-ifeq ($(TARGET_BUILD_VARIANT), eng)
-ifeq ($(SEC_BUILD_OPTION_PRODUCT_SHIP), false)
-ifeq ($(SEC_BUILD_OPTION_SELINUX_ENFORCE),true)
-ccflags-y += -DSEC_SELINUX_DEBUG
-endif
-endif
-endif
-
-ifeq ($(SEC_BUILD_OPTION_VTS),false)
-ifeq ($(TARGET_BUILD_VARIANT), user)
-ifeq ($(SEC_BUILD_OPTION_PRODUCT_SHIP), true)
-ifeq ($(SEC_BUILD_OPTION_SELINUX_ENFORCE),true)
-ccflags-y += -DCONFIG_ALWAYS_ENFORCE=true
-endif
-endif
-endif
-endif
-# ] SEC_SELINUX_PORTING_COMMON
+ccflags-y := -I$(srctree)/security/selinux -I$(srctree)/security/selinux/include
 
 obj-$(CONFIG_SECURITY_SELINUX) := selinux.o
 